internal/config/config.go

```diff
@@ -37,20 +38,52 @@ func defConfig() Config {
 	}
 }
 
-func Load() error {
+func getPath(optionalPath string) (string, error) {
+	// Trigger an error if config flag used but is empty.
+	if len(optionalPath) == 0 {
```

Comment:
Strings are comparable in Go.

```suggestion
	if optionalPath == "" {
```